flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/table/operations/JavaDataStreamQueryOperation.java

```diff
@@ -38,6 +38,11 @@
 @Internal
 public class JavaDataStreamQueryOperation<E> implements QueryOperation {
 
+	// JavaDataStreamQueryOperation represent a registered DataStream from `registerDataStream` method
```

Comment:
These seems more suitable to be commit message but not as comments.

Reply:
what do you think about this?

Reply:
I have added this to commit message. But also keep is as comments.

Reply:
I don't get why we want to keep this as comments? It's just a table name, and due to some reason we lost it in the old code, but added back now. We don't have to record all the reasons behind this in comment.

Reply:
I think it explains clearly why we have this field in it (we don't have this information when initialization but pass in when converting to RelNode). 

But I'm also fine to remove this.